src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,61 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[],
+  setSelectedTodo: React.Dispatch<React.SetStateAction<Todo | null>>,
```

Comment:
```suggestion
  onSelectedTodo: React.Dispatch<React.SetStateAction<Todo | null>>,
```

Reply:
https://jaketrent.com/post/naming-event-handlers-react

Reply:
make sure to fix it all over your code

Reply:
++
thanks